docs/source/redhat-opens-install.rst

```diff
@@ -494,6 +496,12 @@ On each compute node, perform the following steps:
     generate the configuration for a single peer connection, which you can
     duplicate and update for each compute host in your mesh.
 
+    To maintain connectivity between VMs if BIRD crashes or is reloaded, enable
+    BIRD graceful restart by adding `-R` to the `ExecStart` line in the BIRD
+    systemd unit file (`/usr/lib/systemd/system/bird.service` and `bird6.service`).
+    Please not that this will not work if BIRD is stopped normally - if you need
```

Comment:
@matthewdupre As discussed, I think this text needs to say "will not work if BIRD is restarted" and then you need to provide instructions on how to modify upstart accordingly (which you should also test!).